hedera-mirror-rosetta/app/persistence/transaction/transaction_test.go

```diff
@@ -0,0 +1,738 @@
+/*-
+ * ‌
+ * Hedera Mirror Node
+ *
+ * Copyright (C) 2019 - 2020 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+package transaction
+
+import (
+	"encoding/hex"
+	"github.com/DATA-DOG/go-sqlmock"
+	rTypes "github.com/coinbase/rosetta-sdk-go/types"
+	entityid "github.com/hashgraph/hedera-mirror-node/hedera-mirror-rosetta/app/domain/services/encoding"
+	"github.com/hashgraph/hedera-mirror-node/hedera-mirror-rosetta/app/domain/types"
+	"github.com/hashgraph/hedera-mirror-node/hedera-mirror-rosetta/app/errors"
+	"github.com/hashgraph/hedera-mirror-node/hedera-mirror-rosetta/app/persistence/common"
+	"github.com/hashgraph/hedera-mirror-node/hedera-mirror-rosetta/test/mocks"
+	hexutils "github.com/hashgraph/hedera-mirror-node/hedera-mirror-rosetta/tools/hex"
+	"github.com/stretchr/testify/assert"
+	"reflect"
+	"regexp"
+	"strconv"
+	"testing"
+)
+
+var (
+	firstAccount, _          = constructAccount(1)
```

Comment:
imo you should not be using the `constructAccount` function. My thinking is that you are testing f.e `FindBetween` in the `TestShouldSuccessFindBetween` function which under the hood uses the `constructAccount` function and after that you are expecting that the result of the function is equal to the expected output. Basically `constructAccount(x) == `constructAccount(x)`